PageSpeed/Test/NewTestViewController.swift

```diff
@@ -78,6 +79,8 @@ class NewTestViewController: UIViewController {
 
     // MARK: - Methods
     func processRequestsToServices(url: String) {
+        progressView = ProgressView()
+        view.addSubview(progressView!)
```

Comment:
@ilyageekhub you should use local variable for pre-loader and initiate it right before
>DispatchQueue.global(qos: .userInitiated).async {

These lines should be removed.

Reply:
fix